src/app/store.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-useless-return */
+import Task from './task.js';
+
+class Store {
+  static getFromStore() {
+    let taskArray;
+    if (localStorage.getItem('tasks') === null) {
+      taskArray = [];
+    } else {
+      taskArray = JSON.parse(localStorage.getItem('tasks'));
+    }
+    return taskArray;
+  }
+
+  static addToStore(task) {
+    const newTaskArray = Store.getFromStore();
+    newTaskArray.push(task);
+    localStorage.setItem('tasks', JSON.stringify(newTaskArray));
+  }
+
+  static updateToStore(e) {
+    const tasks = Store.getFromStore();
+    const newTask = new Task();
+    const taskIndex = e.target.parentElement.firstChild.id;
+    const taskDescription = e.target.value;
+    tasks.forEach((task, position) => {
+      if (task.index === Number(taskIndex)) {
+        newTask.description = taskDescription;
+        newTask.index = task.index;
+        tasks.splice(position, 1, newTask);
+      }
+    });
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
```

Comment:
[Optional]
- [ ] Instead of reloading the page after every update, you can just update the DOM. It is bad practice to reload the page each time. It is disrupting the flow of the application

Reply:
Hello, @sinansevgi  Thank you for the review. :) The optional suggestion was a big problem for me, I'm still looking for better ways. I already try DOMContentLoaded event or did try to call showTasks method on top of everything. That didn't help as I expected yet. Sometimes I get duplicated items and sometimes empty novelists while the page needs to be reloaded. However, I totally agree with the suggestion. Thank you once again 😄

Reply:
You can just remove all child nodes of `dynamicTaskDiv` and re-create them from the local storage. 

For Example :arrow_down: 
```javascript
  static reloadUI() {
      while (dynamicDiv.firstChild) { dynamicDiv.removeChild(dynamicDiv.firstChild); }
      showTasks()
  }
```
And also you may want to change the `form` container to a `div` container. ( or set an [`onsubmit`](https://www.w3schools.com/jsref/event_onsubmit.asp) event ) By default, the form container refreshes the page when you submit something.